devel/hub/Portfile

```diff
@@ -39,18 +40,18 @@ post-extract {
 }
 
 destroot {
-    xinstall -W ${worksrcpath} bin/hub ${destroot}${prefix}/bin
+    xinstall -m 0755 -W ${worksrcpath} bin/hub ${destroot}${prefix}/bin
     set man-pages ${destroot}${prefix}/share/man/man1
     xinstall -d ${man-pages}
     foreach m [glob -directory ${worksrcpath}/share/man/man1 *.1] {
-        xinstall -W ${worksrcpath} ${m} ${man-pages}/
+        xinstall -m 0644 -W ${worksrcpath} ${m} ${man-pages}/
     }
     set bash-completions ${destroot}${prefix}/share/bash-completion/completions
     xinstall -d ${bash-completions}
-    xinstall -W ${worksrcpath} etc/hub.bash_completion.sh ${bash-completions}/hub
+    xinstall -m 0644 -W ${worksrcpath} etc/hub.bash_completion.sh ${bash-completions}/hub
     set site-functions ${destroot}${prefix}/share/zsh/site-functions
     xinstall -d ${site-functions}
-    xinstall -W ${worksrcpath} etc/hub.zsh_completion ${site-functions}/
+    xinstall -m 0644 -W ${worksrcpath} etc/hub.zsh_completion ${site-functions}/_hub
```

Comment:
I only just noticed this is now being copied to `_hub` where before it didn't have its name changed. Is this correct?

Reply:
Yes. This is what I really want commit, not those `chmod`. A zsh completion file should starts with `_`, otherwise it's not used as completion file. The completion for `hub` in zsh didn't work before this commit.

[Quote from zsh-completions plugin](https://github.com/zsh-users/zsh-completions/blob/master/zsh-completions-howto.org#telling-zsh-which-function-to-use-for-completing-a-command)
> Completion functions for commands are stored in files with names beginning with an underscore `_`, and these files should be placed in a directory listed in the `$fpath` variable.

Reply:
@pmetzger Sorry, seems I forgot to emphasize the name change in the description.